### PR TITLE
Add option to pray for help

### PR DESCRIPTION
**Pull request by mcupery**
Please review this, @casey-collab

### CODE REVIEW

File: english/light-fire/succumb/game-over.md [hunk 1]
```diff
@@ -8,3 +8,5 @@ help but think about everything you're leaving behind.
 [Fight the sensation.  Claw back to life.](../../marshmallow.md)
 
 [Time to find out what heaven is like](../../marshmallow.md)
+
+[Pray for help](/get-help/help.md)
```
Comment (casey-collab) [inline english/light-fire/succumb/game-over.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/light-fire/succumb/get-help/help.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+You pray to your chosen deity for help.
```
Comment (casey-collab) [inline english/light-fire/succumb/get-help/help.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/light-fire/succumb/get-help/help.md (hunk 2 above)
Comment (mcupery) [inline english/light-fire/succumb/get-help/help.md]: This should be fixed now, @casey-collab .  Please reeval.

**Comment by mcupery** [comment]
Please review again, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :ribbon:

**Pull request merged**